db/dataGeneration.js

```diff
@@ -27,7 +29,7 @@ const generateCSVData = function (writer, encoding, callback) {
         for (let i = 0; i < reviewCount; i++) {
           let user = usernames[Math.floor(Math.random() * 8)];
           let score = digits[i];
-          let title = faker.random.words();
+          let title = titles[i];
```

Comment:
maybe I'm missing something, but wouldn't this run out of titles really quickly? I think I would've made this `titles[i % titles.length]`

Same comment applies to `score` and  `recommend`

Reply:
Oh I see what you are saying. The reason why it works is because I am creating 12 reviews per product, and my arrays have 12 elements. So every time a new product is created, the for loop starts again from index 0 and goes up to 12 and then another product is created. So I never end up running out of titles because I only use 12 reviews titles per product.